controller/src/conf/controller.config.properties

```diff
@@ -68,6 +68,13 @@ controller.retention.frequency.minutes=${RETENTION_FREQUENCY_MINUTES}
 controller.retention.bucket.count=${BUCKET_COUNT}
 controller.retention.thread.count=${RETENTION_THREAD_POOL_SIZE}
 
+controller.retention.type=${RETENTION_TYPE}
```

Comment:
Having no "global" retention policy means that by default, we do not apply retention to streams and that each stream that is created will get its data stored forever. Therefore, if a user wants to truncate a stream based on time or size, is the user that can update the stream policy with the appropriate retention configuration.

Reply:
This will give the user an extra advantage for setting the retention policy globally once and free for the rest. by default the flag will remain false but if the user wants to set up some generic retention policy for all streams then the user can set the config values and the system will read that during the creation.
However, if the `StreamConfiguration` is already having some `RetentionPolicy` set, then that takes priority